### PR TITLE
Add travel_type

### PR DESCRIPTION
**Pull request by dgearhart**

### CODE REVIEW

File: proto/tripdirections.proto [hunk 1]
```diff
@@ -84,7 +72,41 @@ message TripDirections {
     kBicycle = 2;
     kTransit = 3;
   }
-    
+
+  // TODO: review and update as needed
+  enum VehicleType {
```
Comment (kevinkreiser) [inline proto/tripdirections.proto]: i get that, for the sake of organization, having these separate is nice but it makes the code very messy, always having to check which type it is to make sure to use the right set of enums. IMHO this lines up more with the having the `travel_type` under the `*_info` sub tree in the response which i thought we decided not to do: https://github.com/valhalla/tyr/pull/111/files#diff-889ae3d00199d1801bdedb1bd876aaf5R758

in other words, why not just organize it like the response is going to be? i dont feel strongly about this at all so just ignore me if that is more convenient :smile:

File: proto/tripdirections.proto [hunk 2]
```diff
@@ -53,17 +53,6 @@ message TripDirections {
   }
 
   message TransitInfo {
-    enum Type {
-      kTram = 0;
-      kMetro = 1;
-      kRail = 2;
-      kBus = 3;
-      kFerry = 4;
```
Comment (dgearhart) [inline proto/tripdirections.proto]: When we discussed yesterday we decided to leave them separate in the protobuf
